AirCasting/Utils/Connection/Connectable.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+protocol Connectable {
```

Comment:
I like the name, but not for this one. This protocol is not for connecting per se, just for checking if a connection can be made. I think we need a better suiting name... Can't think of any good one atm tho, sorry 😅